Dockerfile-alpine.template

```diff
@@ -49,13 +50,11 @@ RUN set -ex \
     && cd src/gdal-${GDAL_VERSION} \
     && ./configure --with-python --with-curl --with-java=${JAVA_HOME} --with-openjpeg=$ROOTDIR \
     && make -j3 && make -j3 install \
-    # Compile Python and Java bindings for GDAL
+    # Compile Python bindings for GDAL
```

Comment:
I made this comment before I fully understood what removing lines 57-8 meant -- we are in fact still installing Java bindings in this block of code, we just aren't moving the shared objects over manually. I can revert this change if it's confusing.